ext/fg/js/popup.js

```diff
@@ -201,6 +203,9 @@ class Popup {
     _inject() {
         if (this._injectPromise === null) {
             this._injectPromise = this._createInjectPromise();
+            this._injectPromise.then(
+                () => { this._injectPromiseComplete = true; },
+                () => { this._resetFrame(); });
```

Comment:
Should the closing paren be on a new line? Don't mind this too much but I've seen that kind of style more.

Reply:
Oops again; yes it should. Different language styles mixing me up.

Reply:
Style fixed in b13957ea8efa58bd2d88615262282fecad151f61.

Reply:
Are you secretly writing Lisp somewhere )))

Reply:
The new style looks good :+1:

Reply:
Thankfully no; it stems from whether languages/conventions prefer brackets on the same line as the expression, or on a new line.

```js
// Same line
if (
    something1 &&
    something2
) {
    // ...
}

// New line
if (
    something1 &&
    something2)
{
    // ...
}

// Yuck
if (
    something1 &&
    something2
)
{
    // ..
}
```

Reply:
Ok, I've seen lots of C code to put brackets on a new line. Java/JS/PHP etc seems to prefer the same line. Not sure how I'd handle multi line ifs with the new line style. Maybe I'd create a variable for the boolean before testing it in more complex cases.